quarkchain/cluster/cluster.py

```diff
@@ -31,12 +31,15 @@ def kill_child_processes(parent_pid, sig=signal.SIGTERM):
         process.send_signal(sig)
 
 
-async def run_master(config_file, check_db, profile):
+async def run_master(config_file, check_db, profile, rblock_from, rblock_to):
```

Comment:
prefer having `rblock_from` and `rblock_to` together as a string arg or dict arg

Reply:
Done.